### PR TITLE
W-16224232 runtime logs gr

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.8/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -95,7 +95,7 @@
 *** xref:flex-view-api-metrics.adoc[View Key Metrics for Flex Gateway APIs]
 *** xref:flex-view-replica-status.adoc[View the Replica Status in Runtime Manager]
 *** xref:flex-view-api-status.adoc[View the API Status in Runtime Manager]
-*** xref:flex-view-logs-in-monitoring.adoc[View Logs in Anypoint Monitoring]
+*** xref:flex-view-logs-in-runtime-manager.adoc[]
```
Comment (chris-campbell9) [inline gateway/1.8/modules/ROOT/nav.adoc]:
```suggestion
*** xref:flex-view-logs-in-runtime-manager.adoc[View Logs in Runtime Manager]
```
Consistency with the other imperative page titles. Leaving empty brackets will result in a title with a gerund. This requirement goes back to the days of Kim. If we want all our page titles to contain gerunds instead, we'll need to change most if not all nav page titles.

File: gateway/1.8/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc [hunk 2]
```diff
@@ -6,8 +6,9 @@ The following configuration file examples demonstrate how to direct Flex Gateway
 //end::intro1[]
 
 //tag::intro2-conn[]
-As well as being able to configure additional Fluent Bit log output types, Flex Gateway sends runtime logs to runtime manager and access logs to API Manager. Use the logs for quick log checks when an additional log output is not needed. To view runtime logs, see xref:flex-view-logs-in-monitoring.adoc[], and to configure and view access logs, see xref:flex-conn-message-log.adoc[]. For access logs to be visible in *Logs*, you must additionally apply a Message Logging policy.
+As well as being able to configure additional Fluent Bit log output types, Flex Gateway sends runtime and access logs to runtime manager. To view logs, see xref:flex-view-logs-in-runtime-manager.adoc[].
```
Comment (chris-campbell9) [inline gateway/1.8/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc]:
```suggestion
As well as being able to configure additional Fluent Bit log output types, Flex Gateway sends runtime and access logs to Runtime Manager. To view logs, see xref:flex-view-logs-in-runtime-manager.adoc[]. 
```

File: gateway/1.8/modules/ROOT/pages/flex-conn-monitor.adoc [hunk 3]
```diff
@@ -14,5 +14,5 @@ Refer to the following tasks for details about monitoring in Connected Mode:
 * xref:flex-view-api-metrics.adoc[View Key Metrics for Flex Gateway APIs] - Collect metrics for active Flex Gateway APIs via API Manager. You can view charts and filter metrics by date.
 * xref:flex-view-replica-status.adoc[View the Replica Status in Runtime Manager] - View Flex Replica connection status from within the Runtime Manager UI.
 * xref:flex-view-api-status.adoc[View the Flex Gateway API Status in Runtime Manager] - View the Flex Gateway API status from within the Runtime Manager UI.
-* xref:flex-view-logs-in-monitoring.adoc[View Logs in Anypoint Monitoring] - View details of your Flex Gateway logs from within the Anypoint Monitoring UI.
+* xref:flex-view-logs-in-runtime-manager.adoc[View Logs in Anypoint Monitoring] - View details of your Flex Gateway logs from within the Anypoint Monitoring UI.
```
Comment (chris-campbell9) [inline gateway/1.8/modules/ROOT/pages/flex-conn-monitor.adoc]:
```suggestion
* xref:flex-view-logs-in-runtime-manager.adoc[View Logs in Runtime Manager] - View details of your Flex Gateway logs from within the Runtime Manager UI.
```

File: gateway/1.8/modules/ROOT/pages/flex-troubleshoot-debug-logs.adoc [hunk 4]
```diff
@@ -36,5 +36,5 @@ If you have already configured a logging configuration file, set `logging.runtim
 
 To view runtime log output, see:
 
-* xref:flex-view-logs-in-monitoring.adoc[View Logs in Connected Mode]
+* xref:flex-view-logs-in-runtime-manager.adoc[View Logs in Connected Mode]
```
Comment (chris-campbell9) [inline gateway/1.8/modules/ROOT/pages/flex-troubleshoot-debug-logs.adoc]:
```suggestion
* xref:flex-view-logs-in-runtime-manager.adoc[View Logs in Runtime Manager]
```

File: gateway/1.8/modules/ROOT/pages/flex-view-logs-in-runtime-manager.adoc [hunk 5]
```diff
@@ -0,0 +1,23 @@
+= Viewing Logs in Runtime Manager
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: runtime-manager::flex-gateway-monitoring.adoc, flex-view-logs-in-monitoring.adoc
+:imagesdir: ../assets/images
+
+Use Runtime Manager to view the runtime and access logs of your Flex Gateway. Runtime Manager enables you search for key words within the logs and to filter by *Replica*, *Time Range*, and *Log Level*.
```
Comment (chris-campbell9) [inline gateway/1.8/modules/ROOT/pages/flex-view-logs-in-runtime-manager.adoc]:
```suggestion
Use Runtime Manager to view the runtime and access logs of your Flex Gateway. Runtime Manager enables you to search for keywords within the logs and to filter by *Replica*, *Time Range*, and *Log Level*.
```

File: gateway/1.8/modules/ROOT/pages/flex-view-logs-in-runtime-manager.adoc [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+NOTE: For an API's access logs to be visible, you must apply a Message Logging policy to that API. To learn more, see xref:flex-conn-message-log.adoc[].
+
+To access Flex Gateway logs in the Runtime Manager:
```
Comment (chris-campbell9) [inline gateway/1.8/modules/ROOT/pages/flex-view-logs-in-runtime-manager.adoc]:
```suggestion
To access Flex Gateway logs in Runtime Manager:
```

File: gateway/1.8/modules/ROOT/pages/flex-view-logs-in-runtime-manager.adoc [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Flex Gateway* in the navigation menu.
+. Click the name of the gateway to which you want to add an API.
```
Comment (chris-campbell9) [inline gateway/1.8/modules/ROOT/pages/flex-view-logs-in-runtime-manager.adoc]: I'm a little confused. Are they adding an API here? Or are they ultimately viewing logs in Runtime Manager?

File: gateway/1.8/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc [hunk 8]
```diff
@@ -6,8 +6,9 @@ The following configuration file examples demonstrate how to direct Flex Gateway
 //end::intro1[]
 
 //tag::intro2-conn[]
-As well as being able to configure additional Fluent Bit log output types, Flex Gateway sends runtime logs to runtime manager and access logs to API Manager. Use the logs for quick log checks when an additional log output is not needed. To view runtime logs, see xref:flex-view-logs-in-monitoring.adoc[], and to configure and view access logs, see xref:flex-conn-message-log.adoc[]. For access logs to be visible in *Logs*, you must additionally apply a Message Logging policy.
+As well as being able to configure additional Fluent Bit log output types, Flex Gateway sends runtime and access logs to Runtime Manager. To view logs, see xref:flex-view-logs-in-runtime-manager.adoc[]. 
 
+To enable access logs for an API, you must apply a message logging policy. To learn more, see xref:flex-conn-message-log.adoc[].
```
Comment (chris-campbell9) [inline gateway/1.8/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc]:
```suggestion
To enable access logs for an API, you must apply a Message Logging policy. To learn more, see xref:flex-conn-message-log.adoc[].
```

File: gateway/1.8/modules/ROOT/pages/flex-conn-third-party-logs-config.adoc [hunk 9]
```diff
@@ -39,7 +39,7 @@ To configure access logs in your log output, you must apply a Message Logging po
 
 To configure a Message Logging policy, see xref:flex-conn-message-log.adoc[].
 
-Configuring a Message Logging policy makes access logs visible in both API Manager *Logs* and your third-party log services.
+Configuring a Message Logging policy makes access logs visible in API Manager, Runtime Manager, and your third-party log services.
```
Comment (chris-campbell9) [inline gateway/1.8/modules/ROOT/pages/flex-conn-third-party-logs-config.adoc]: Should API Manager still be mentioned? You removed it from other pages. If customers can still view logs in API Manager then it should be included in the other pages.

File: gateway/1.8/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc [hunk 10]
```diff
@@ -6,8 +6,9 @@ The following configuration file examples demonstrate how to direct Flex Gateway
 //end::intro1[]
 
 //tag::intro2-conn[]
-As well as being able to configure additional Fluent Bit log output types, Flex Gateway sends runtime logs to runtime manager and access logs to API Manager. Use the logs for quick log checks when an additional log output is not needed. To view runtime logs, see xref:flex-view-logs-in-monitoring.adoc[], and to configure and view access logs, see xref:flex-conn-message-log.adoc[]. For access logs to be visible in *Logs*, you must additionally apply a Message Logging policy.
+As well as being able to configure additional Fluent Bit log output types, Flex Gateway sends runtime and access logs to Runtime Manager. To view logs, see xref:flex-view-logs-in-runtime-manager.adoc[].
```
Comment (seblaz) [inline gateway/1.8/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc]: Actually, Flex sends all logs to Monitoring. Then logs can be seen from Monitoring, Runtime Manager or Api Manager

File: gateway/1.8/modules/ROOT/pages/flex-view-logs-in-runtime-manager.adoc [hunk 11]
```diff
@@ -0,0 +1,24 @@
+= Viewing Logs in Runtime Manager
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: runtime-manager::flex-gateway-monitoring.adoc, flex-view-logs-in-monitoring.adoc
+:imagesdir: ../assets/images
+
+Use Runtime Manager to view the runtime and access logs of your Flex Gateway. Runtime Manager enables you to search for keywords within the logs and to filter by *Replica*, *Time Range*, and *Log Level*.
+
+NOTE: For an API's access logs to be visible, you must apply a Message Logging policy to that API. To learn more, see xref:flex-conn-message-log.adoc[].
```
Comment (ahejeij) [inline gateway/1.8/modules/ROOT/pages/flex-view-logs-in-runtime-manager.adoc]: Runtime manager logs are runtime logs only, no access logs

File: gateway/1.8/modules/ROOT/pages/flex-conn-third-party-logs-config.adoc [hunk 12]
```diff
@@ -39,8 +39,6 @@ To configure access logs in your log output, you must apply a Message Logging po
 
 To configure a Message Logging policy, see xref:flex-conn-message-log.adoc[].
 
-Configuring a Message Logging policy makes access logs visible in both API Manager *Logs* and your third-party log services.
-
```
Comment (ahejeij) [inline gateway/1.8/modules/ROOT/pages/flex-conn-third-party-logs-config.adoc]: I would not remove this line

File: gateway/1.8/modules/ROOT/pages/flex-view-logs-in-runtime-manager.adoc [hunk 13]
```diff
@@ -0,0 +1,24 @@
+= Viewing Logs in Runtime Manager
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: runtime-manager::flex-gateway-monitoring.adoc, flex-view-logs-in-monitoring.adoc
+:imagesdir: ../assets/images
+
+Use Runtime Manager to view the runtime logs of your Flex Gateway. Runtime Manager enables you to search for keywords within the logs and to filter by *Replica*, *Time Range*, and *Log Level*.
```
Comment (ahejeij) [inline gateway/1.8/modules/ROOT/pages/flex-view-logs-in-runtime-manager.adoc:8]: I would add the same warning that we have on the access logs page, that states the storage limits of 100mb or 30 days

Comment (ahejeij) [reply gateway/1.8/modules/ROOT/pages/flex-view-logs-in-runtime-manager.adoc:8]: here https://docs.mulesoft.com/gateway/latest/flex-conn-message-log, the limits and features section

File: gateway/1.8/modules/ROOT/pages/flex-view-logs-in-runtime-manager.adoc [hunk 14]
```diff
@@ -0,0 +1,26 @@
...
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: runtime-manager::flex-gateway-monitoring.adoc, flex-view-logs-in-monitoring.adoc
+:imagesdir: ../assets/images
+
+Use Runtime Manager to view the runtime logs of your Flex Gateway. Runtime Manager enables you to search for keywords within the logs and to filter by *Replica*, *Time Range*, and *Log Level*. 
+
+Access logs are not visible in Runtime Manager. To learn more about access logs, see xref:flex-conn-message-log.adoc[].
+
+Runtime Manager stores up to 100 MB of logs per Flex Replica for 30 days. Runtime Manager automatically purges logs that go beyond 100 MB per Flex Replica. If you need more storage, you can either use a third-party service or sign up for the Anypoint Integration Advanced package or a Titanium subscription to Anypoint Platform.
```
Comment (ahejeij) [inline gateway/1.8/modules/ROOT/pages/flex-view-logs-in-runtime-manager.adoc]: replace `per Flex Replica` with `per Flex Gateway`. Also remove Integration Advanced package, leave only Titanium subscriptions

**Review by failup — APPROVED** [review]

File: gateway/1.8/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc [hunk 15]
```diff
@@ -6,10 +6,12 @@ The following configuration file examples demonstrate how to direct Flex Gateway
 //end::intro1[]
 
 //tag::intro2-conn[]
-As well as being able to configure additional Fluent Bit log output types, Flex Gateway sends runtime logs to runtime manager and access logs to API Manager. Use the logs for quick log checks when an additional log output is not needed. To view runtime logs, see xref:flex-view-logs-in-monitoring.adoc[], and to configure and view access logs, see xref:flex-conn-message-log.adoc[]. For access logs to be visible in *Logs*, you must additionally apply a Message Logging policy.
+As well as being able to configure additional Fluent Bit log output types, Flex Gateway sends runtime logs to runtime manager and access logs to API Manager. To learn more, see:
```
Comment (chris-campbell9) [inline gateway/1.8/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc:9]: Capitalize "Runtime Manager".

**Pull request merged**